be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/ElectionHandler.scala

```diff
@@ -54,7 +54,7 @@ class ElectionHandler(dbRef: => AskableActorRef) extends MessageHandler {
       _ <- dbActor ? DbActor.WriteAndPropagate(rpcMessage.getParamsChannel, message)
       _ <- dbActor ? DbActor.CreateChannel(electionChannel, ObjectType.ELECTION)
       _ <- dbActor ? DbActor.WriteAndPropagate(electionChannel, message)
-      _ <- dbActor ? DbActor.CreateElectionData(electionId, keyPair)
+      _ <- dbActor ? DbActor.CreateElectionData(rpcMessage.extractLaoId, electionId, keyPair)
```

Comment:
`extractLaoId` above may or may not need to be changed depending on pr #1459, I am not sure if here the actual laoId is really retrieved, gotta be careful

Reply:
A Setup Election message is sent on the main channel `root/laoId`, so here by applying `extractLaoId` we actually retrieve the laoId. 
It is the only exception among the election messages. Otherwise, we should be careful with the other messages.